tasks/main.yml

```diff
@@ -3,3 +3,5 @@
 - import_tasks: pre_checks.yml
 
 - include_tasks: state_{{ (k3s_state | lower) | default('installed') }}.yml
+
+- include_tasks: download_kubeconfig.yml
```

Comment:
:pick: 

 - Newline missing at end of file.
 - Move `when: k3s_download_kubeconf` here.
 - The imported tasks are not idempotent.

Reply:
Could you check? I think it's solved